src/Worker/MergeContact.php

```diff
@@ -52,6 +52,7 @@ public static function execute(int $new_cid, int $old_cid, int $uid)
 		// These fields only contain public contact entries (uid = 0)
 		if ($uid == 0) {
 			DBA::update('post-tag', ['cid' => $new_cid], ['cid' => $old_cid]);
+			DBA::delete('post-tag', ['cid' => $old_cid]);
```

Comment:
Isn't this supposed to delete exactly 0 rows after the previous query?

Reply:
No. Sometimes the `new_cid` and `old_cid` are both used in the same post. Since we use an unique key here, for these rows the `cid` won't be updated. So at the end we get rid of them.